react-example/src/config/defaults.js

```diff
@@ -0,0 +1,3 @@
+export const WOOTRIC_ACCOUNT_TOKEN = 'NPS-3bbb4510'; // This is your unique account token.
```

Comment:
Change to fake token...
```suggestion
export const WOOTRIC_ACCOUNT_TOKEN = 'NPS-XXXXXX'; // This is your unique account token.
```